heartbeat/heartbeat.yml

```diff
@@ -128,7 +128,13 @@ output.elasticsearch:
 
 #================================ Processors =====================================
 processors:
-  - add_observer_metadata: ~
+  - add_observer_metadata: 
+# Optional, but recommended geo settings for the location heartbeat is running in 
+#    geo: 
```

Comment:
Hm, that works too. The more standard way we did it so far is to have `#geo`, see lines above. No strong opinion but we should consistent I think.